ui/prometheus-plugin/src/plugins/prometheus-variables.tsx

```diff
@@ -27,6 +27,7 @@ function PrometheusLabelValuesVariableEditor(props: OptionsEditorProps<Prometheu
       <TextField
         sx={{ mb: 1 }}
         label="Label Name"
+        required
         value={props.value.label_name}
         onChange={(e) => {
           props.onChange({ ...props.value, label_name: e.target.value });
```

Comment:
One part of this that I'm still confused about is are any of the example dashboards in [dev/data/dashboard.json](https://github.com/perses/perses/blob/v0.29.1/dev/data/dashboard.json) actually invalid? I know I added a some of the examples in Benchmark and Demo, but is what's inside variables array there out of date?

Reply:
the problem was more when you created a static list, the list was created using the definition `{value: string, label: string}[]` which wasn't known by the cue schema. I just fixed that.

I took the opportunity also to add the `required` attribute where it was necessary. But the form is still valid if you don't fill the mandatory field.